### DISCUSSION TITLE
Error when entering API Key From Click

### DISCUSSION THREAD

**Discussion by HomeField145345** [gwleuverink/clickup-time-tracker#35] [Q&A]
I am getting an error when trying to add the API Key provided by Click it seems its not long enough

**Comment by gwleuverink** [comment]
Hi @ckrablin, I've put minimum character validation on the access token for 43 characters. 

When building this I tested it a couple of times and every time I got a token with a character count of 43. Can you confirm you are using a access token, not oauth credentials? (Check the readme on how to generate an access token)

If you are in fact using the access token please let me know the length of the token you are using and I'll update the code accordingly

**Comment by gwleuverink** [comment]
Hi @ckrablin, do you still experience this?

**Comment by isa-stewart** [comment]
I'm getting an error saying "Please input your ClickUp Access Token" with a 41 character access token.

**Comment by zanga7** [comment]
Im having the same issue with it not accepting my API key. I have 42 characters

**Comment by zanga7** [comment]
I added a space at the end (to make it 43 characters) and it worked.
